scripts/esxi/sysprep.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/bin/sh
 set -euxo pipefail
```

Comment:
Be sure to check the shell-linter output (`make lint`), as this appears to use bash specific directives.